doc/source/mast_data_products/pipeline-astrometric-calibration-description.rst

```diff
@@ -699,22 +699,25 @@ considered compromised. These checks include:
     exposures in the association.
 
 
-  * **radial_offset_check**: Ensures that the image alignment source magnitudes 
-    and astrometric catalog magnitudes have a *fit_rms* greater than 0.5.
-
+  * **radial_offset_check**: Ensures that the radial offset between the 
+    image alignment sources and astrometric catalog sources is reasonable. The 
+    radial offset is computed as the square root of shifts in the tangent plane
+    squared; these shifts are calculated in tweakwcs. This radial offset 
+    (in arcseconds) divided by 10 and raised to the 8th power (+ 0.8), must be 
+    more than the number of cross matches (+ 2) times 0.36 (see align.py).
```

Comment:
Change: must be _more_ than the number of cross matches -> must be _less_ than the number of cross matches

Reply:
Good catch, sorry for the confusion.